internal/pkg/template/templates/workloads/partials/cf/service-base-properties.yml

```diff
@@ -33,11 +33,13 @@ DeploymentConfiguration:
       {{- end }}
     {{- end }}
     Enable: true
-  {{- else }}
-    AlarmNames: []
-    Enable: false
-  {{- end}}
     Rollback: true
+  {{- else }}
+    !If
+    - IsGovCloud
+    - !Ref AWS::NoValue
+    - Enable: false
```

Comment:
Would the non-govcloud users need the `AlarmNames: []` line?

Reply:
yes, definitely. I think it shouldn't matter for functionality if Enable is false but this is cleaner.